mbtools/html_to_json.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import glob
+import json
+from pathlib import Path
+
+
+def create_json_content(uuid, content, variant="main"):
+    json_content = {"id": uuid, "content":
+                    [{"variant": variant, "html": content}]}
+    return json.dumps(json_content, indent=4)
```

Comment:
I realize this is somewhat of a subjective preference, but could we use an indent of 2 instead of 4?